supertokens_python/recipe/session/framework/django/asyncio/__init__.py

```diff
@@ -27,6 +29,12 @@ def verify_session(
     anti_csrf_check: Union[bool, None] = None,
     session_required: bool = True,
     user_context: Union[None, Dict[str, Any]] = None,
+    override_global_claim_validators: Optional[
+        Callable[
+            [SessionContainer, List[SessionClaimValidator], Dict[str, Any]],
```

Comment:
This was flipped as well (and everywhere else we have the `override_global_claim_validators` callback)

Reply:
Just like the node PR, this has been done in a seperate PR 

https://github.com/supertokens/supertokens-python/pull/211